Open-Attestation/open-attestation
# feat: align against W3C VC 2.0 test suite and complete unit tests

## What does this PR do?

**Schema related changes**
- Align against W3C VC 2.0 test suite
  - Utilise `jsonld` for expansion (i.e. ensure VC contexts are interpretable)
  - Utilise `zod` for data model validation
- Avoid using sub types (e.g. `renderMethodType: EMBEDDED_RENDERER`) and just utilise a scoped context (e.g. `type: OpenAttestationEmbeddedRenderer`)
- Make `renderMethod` an array and rename ~~`name`~~ to `templateName`
- Update v4 test fixtures

**Improvements**
- Utilise `cross-fetch` instead due to segmentation fault when using default `nodeDocumentLoader()`
- Digest to handle falsy values correctly

## Prerequisite

- [x] Open-Attestation/schemata/pull/83 has to be merged and deployed first
